Assets/Mirror/Runtime/NetworkConnection.cs

```diff
@@ -262,7 +262,41 @@ internal virtual bool SendBytes(byte[] bytes, int channelId = Channels.DefaultRe
                 return false;
             }
 
-            return TransportSend(channelId, bytes);
+            // send to client or server
+            if (Transport.activeTransport.ClientConnected())
+            {
+                return Transport.activeTransport.ClientSend(channelId, bytes);
+            }
+            else if (Transport.activeTransport.ServerActive())
+            {
+                return Transport.activeTransport.ServerSend(connectionId, channelId, bytes);
+            }
+            return false;
+        }
+
+        // SendBytes to many. basically Transport.Send(connections) + checks.
+        internal static bool SendBytes(List<int> connectionIds, byte[] bytes, int channelId = Channels.DefaultReliable)
+        {
+            if (bytes.Length > Transport.activeTransport.GetMaxPacketSize(channelId))
+            {
+                Debug.LogError("NetworkConnection.SendBytes cannot send packet larger than " + Transport.activeTransport.GetMaxPacketSize(channelId) + " bytes");
+                return false;
+            }
+
+            if (bytes.Length == 0)
+            {
+                // zero length packets getting into the packet queues are bad.
+                Debug.LogError("NetworkConnection.SendBytes cannot send zero bytes");
+                return false;
+            }
+
+            // only the server sends to many, we don't have that function on a
+            // client.
+            if (Transport.activeTransport.ServerActive())
+            {
+                return Transport.activeTransport.ServerSend(connectionIds, channelId, bytes);
```

Comment:
This will not work.

Not all connections are handled by the activeTransport.

In particular, in host mode connection 0 is the local connection and the transport knows nothing about it.

Reply:
this is just an extra code path based on the previous Send(connId), just with a list.
the logic is exactly the same.
are you sure this won't work?
we use if active then serversend right now too :)